src/main/steps/living-england-wales-twelve-months/content.ts

```diff
@@ -0,0 +1,48 @@
+import { YesOrNo } from '../../app/case/case';
+import { TranslationFn } from '../../app/controller/GetController';
+import { FormContent } from '../../app/form/Form';
+import { isFieldFilledIn } from '../../app/form/validation';
+
+const en = ({ required }) => ({
+  title: 'Have you been living in England or Wales for the last 12 months?',
+  errors: {
+    livingInEnglandWalesTwelveMonths: {
+      required,
+    },
+  },
+});
+
+const cy: typeof en = ({ required }) => ({
+  ...en({ required }),
+});
+
+export const form: FormContent = {
+  fields: {
+    livingInEnglandWalesTwelveMonths: {
+      type: 'radios',
+      classes: 'govuk-radios--inline',
+      label: l => l.title,
```

Comment:
You might need to add a `labelHidden` prop here to stop the title being duplicated with the input label.

```suggestion
      label: l => l.title,
      labelHidden: true,
```